brick/__brick__/{{project_name.snakeCase()}}/android/app/build.gradle

```diff
@@ -49,7 +49,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "{{org_name.dotCase()}}.{{project_name.snakeCase()}}"
+        applicationId "{{application_id_android}}"
```

Comment:
We shouldn't be manually changing the contents of the `__brick__` folder by hand since this is all autogenerated.

Reply:
I have made the changes on the project source, and run the generator to make the changes on the `__brick__` folder.

Let me know if this is correct now.